### DISCUSSION TITLE
Installation of Plugin ImportBundle fails

### DISCUSSION THREAD

**Discussion by fibretch** [kimai/kimai#4634] [Plugins]
### Describe the issue

Hi, just tried to install the Plugin ImportBundle. Followed the instructions, but when I reload kimai, it fails with the following message:

```
PHP Fatal error:  Uncaught Error: Class "App\Validator\Constraints\TimesheetDeactivated" not found in /var/www/kimai2/var/plugins/ImportBundle/ImportBundle.php:25
Stack trace:
#0 /var/www/kimai2/vendor/symfony/http-kernel/Kernel.php(446): App\Kernel->registerBundles()
#1 /var/www/kimai2/vendor/symfony/http-kernel/Kernel.php(133): Symfony\Component\HttpKernel\Kernel->initializeBundles()
#2 /var/www/kimai2/vendor/symfony/framework-bundle/Console/Application.php(169): Symfony\Component\HttpKernel\Kernel->boot()
#3 /var/www/kimai2/vendor/symfony/framework-bundle/Console/Application.php(75): Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands()
#4 /var/www/kimai2/vendor/symfony/console/Application.php(149): Symfony\Bundle\FrameworkBundle\Console\Application->doRun()
#5 /var/www/kimai2/bin/console(29): Symfony\Component\Console\Application->run()
#6 {main}
  thrown in /var/www/kimai2/var/plugins/ImportBundle/ImportBundle.php on line 25
```

Any suggestions whats wrong here? Thank you!

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.11

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.2

**Comment by kevinpapst** [comment]
The error message does not make any sense to me, maybe some composer issue
